modules/ROOT/pages/design-edit-mulitple-tabs.adoc

```diff
@@ -0,0 +1,37 @@
+= Edit in Multiple Tabs in the Text Editor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can have multiple files open at a time in API Designer's text editor.
+
+== Opening Files in New Tabs
+
+There are four ways that you can open a file in a new tab:
```

Comment:
I think this needs update as we are documenting three ways (not four)

Reply:
@rddheath I think the "navigate from errors panel" case is missing, right?